src/components/Settings/SettingsContent.jsx

```diff
@@ -14,38 +14,93 @@ const SettingsContent = ({
   onClaudy,
   isDrawer = false,
   isClaudyLoading,
-  toggleSupport,
   shoulDisplayViewOfferButton,
-  managerUrlPremiumLink,
-  viewOfferButtonText
+  managerUrlPremiumLink
 }) => (
   <div className="coz-nav-pop-content">
     {isDrawer && <hr />}
     {settingsAppURL && (
-      <ul className="coz-nav-group">
-        <li className="coz-nav-settings-item">
-          <a
-            role="menuitem"
-            href={`${settingsAppURL}#/profile`}
-            target="_self"
-            data-icon="icon-profile"
-            title={t('profile')}
-          >
-            <p className="coz-label">{t('profile')}</p>
-          </a>
-        </li>
-        <li className="coz-nav-settings-item">
-          <a
-            role="menuitem"
-            href={`${settingsAppURL}#/connectedDevices`}
-            target="_self"
-            data-icon="icon-connectedDevices"
-            title={t('connectedDevices')}
-          >
-            <p className="coz-label">{t('connectedDevices')}</p>
-          </a>
-        </li>
-      </ul>
+      <>
+        <ul className="coz-nav-group">
+          <li className="coz-nav-settings-item">
+            <a
+              role="menuitem"
+              href={`${settingsAppURL}#/profile`}
+              target="_self"
+              data-icon="icon-profile"
+              title={t('profile')}
+            >
+              <p className="coz-label">{t('profile')}</p>
+            </a>
+          </li>
+          {(!isDrawer || !isMobileApp()) && shoulDisplayViewOfferButton && (
+            <li className="coz-nav-settings-item">
+              <a
+                role="menuitem"
+                href={managerUrlPremiumLink}
+                target="_self"
+                data-icon="icon-cozy-circle"
+                title={t('plans')}
+              >
+                <p className="coz-label">{t('plans')}</p>
+              </a>
+              <span data-icon="icon-openwith"></span>
+            </li>
+          )}
+          {!isDrawer && storageData && (
+            <>
+              <li className="coz-nav-settings-item">
+                <a
+                  role="menuitem"
+                  data-icon="icon-graph-circle"
+                  target="_self"
+                  title={t('storage')}
+                  href={`${settingsAppURL}#/storage`}
+                  className="coz-nav-settings-item-storage-link"
+                >
+                  {t('storage')}
+                  <StorageData data={storageData} />
+                </a>
+              </li>
+            </>
+          )}
+        </ul>
+        <ul className="coz-nav-group">
+          <li className="coz-nav-settings-item">
+            <a
+              role="menuitem"
+              href={`${settingsAppURL}#/permissions/slug`}
```

Comment:
can you hide this link behind this flag? https://github.com/cozy/cozy-settings/blob/e7ee451bc5feb4540f47a00e0683f247658338a4/src/components/Sidebar.tsx#L80